node/src/components/consensus/era_supervisor.rs

```diff
@@ -1019,6 +1024,12 @@ where
                 .collect(),
             ProtocolOutcome::WeAreFaulty => Default::default(),
             ProtocolOutcome::DoppelgangerDetected => Default::default(),
+            // TODO: Replace the panic with the appropriate shutdown effect once that's possible.
+            ProtocolOutcome::FttExceeded => self
+                .effect_builder
+                .set_timeout(Duration::from_millis(FTT_EXCEEDED_SHUTDOWN_DELAY_MILLIS))
+                .then(|_| async move { panic!("shutting down: too many faulty validators") })
+                .ignore(),
```

Comment:
Still, no solution to do it cleanly, huh?

Reply:
Has been fixed now :)